NEWS

```diff
@@ -11,9 +11,156 @@
+ * Allowed user-based existing data copy when appending to array
+ * Improved time units description
+#### UCM
+ * Added memory hooks to track allocations by CUDA memory mapping functions
+#### Tools
+ * Improved performance by increasing window size for put_bw and add get_bw in ucx_perftest
+ * Added multi-send flag for receive operations in bandwidth benchmarks in ucx_perftest
+ * Improved ucx_perftest uni-directional test with added fence
+ * Improved ucx_perftest batch helper notes
+#### Documentation
+ * Added a section regarding adaptive routing on RoCE
+#### Architecture
+ * Added CPU Model for MI300A
+ * Added Fujitsu ARM specific values to ucx.conf
+ * Added AMD Turin support
+#### Build
+ * Improved compiler error reporting with added flag
+ * Improved coverity script to allow faster turnaround time
+ * Improved Intel Compiler detection and support
+#### GO
+ * Added multi-send flag and user memh support in request params
+#### Packaging
+ * Improved dpkg-buildpackage sample command by using explicit arguments
+### Bug
+#### UCP
+ * Fixed stack overflow in exported rkey unpack
+ * Removed extra remote-cpu overhead from protocol estimation for zcopy
+ * Fixed performance estimation for rndv pipeline protocols
+ * Fixed ATP sending by picking the correct lane
+ * Fixed missing reg_id on memh creation
+ * Fixed repeated invalidations by retaining existing access flags
+ * Fixed abort reason propagation for rendezvous RTR mtype
+ * Fixed transport availability for configuration list in ALLOW mode
+ * Fixed wrong flag being used for checking BCOPY capability
+ * Fixed sending too many ATPs for small messages
+ * Enforced 16 bits size for Active Messages identifiers
+ * Fixed unnecessary status check for emulated AMO
+ * Fixed more than one fragment sending in rendezvous pipeline
+ * Fixed crash by using biggest max frag across all lanes
+ * Fixed missing memory handle flags by copying from parent to child
+ * Fixed worker interface activate count
+ * Fixed flush requests by replacing ATP/flush lane map with lane indexes
+ * Fixed lost uct_flags when merging memory regions
+#### UCT
+ * Fixed memory domain UCT flags description
+#### RDMA CORE (IB, ROCE, etc.)
+ * Fixed FETCH_ADD remote access error for ODP/KSM case
+ * Fixed missing conditional compilation checks for DM
+ * Fixed IB MD allocation naming typo
+ * Fixed invalid GIDs filter in IB
+ * Fixed flags usage in MLX5 zcopy_post
+ * Do not limit ODP registration retries
+ * Fixed JUCX failures by considering the number of supported completion vectors
+#### CUDA
+ * Fixed async memory handling using CUDA memory type on Grace
+ * Added rcache overhead in performance estimation
+ * Fixed GDR copy performance regression by providing maximum estimation between get and put
+ * Fixed CUDA IPC reachability check
+ * Fixed crash in MPI_Finalize when CUDA context is destroyed
+ * Always require rcache by default for GDR
+ * Fixed md_close when no rcache
```

Comment:
expand

Reply:
fixed "Fixed crash GDR copy in UCT MD close when rcache was not enabled"